.github/workflows/blade-validate.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        shard: [1/4, 2/4, 3/4, 4/4]
+        shard: [1, 2, 3, 4]
```

Comment:
I think it needs to be in `(?<shardIndex>\d+)/(?<shardCount>\d+)` format

 https://jestjs.io/docs/next/cli#--shard

Reply:
if you want to used [1, 2, 3, 4] then in set the SHARD env as `--shard=${{ matrix.shard }}/${{ strategy.job-total }}`